benches/rustposix-criterion.rs

```diff
@@ -1,68 +1,157 @@
-use criterion::{black_box, criterion_group, criterion_main, Criterion};
+/* Benchmarks for the microvisor implementation.  In general, I'm not doing
+ * results checking / assertations to avoid adding bias to the results.  */
+
+
+// I hate allowing this, but this is apparently a known issue for a lot of 
```

Comment:
There are some helper functions in tests/mod.rs that I believe would help you do these more safely.

Reply:
I think I addressed this.  I use those functions now.